simp_le.py

```diff
@@ -1511,11 +1511,14 @@ def test_it(self):
         files = ('account_key.json', 'key.pem', 'full.pem')
         with self._new_swd():
             self.assertEqual(EXIT_RENEWAL, self._run(args))
+            # Set time to the past to ensure that changes are detected.
+            for path in files:
+                os.utime(path, (0, 0))
             initial_stats = self.get_stats(*files)
 
             self.assertEqual(EXIT_NO_RENEWAL, self._run(args))
             # No renewal => no files should be touched
-            # NB get_stats() would fail if file didn't exist
+            # NB os.utime and get_stats() would fail if file didn't exist
```

Comment:
`os.utime` is not even used here, this line should be reverted - the comment refers to what would happen affter `_run` is executed